spaceR/spaceR/ViewModels/SignupViewModel.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SignupViewModel.swift
+//  SpaceR
+//
+//  Created by Kieran Woodrow on 2022/03/13.
+//
+
+import Foundation
+import UIKit
+
+class SignupViewModel {
+    
+    private var userFirstName = ""
+    private var userLastName = ""
+    private var userEmail = ""
+    private var userPassword = ""
+    // swiftlint:disable force_cast
```

Comment:
Don't disable the rule 😄 Rather just handle this a little bit more safely, right now you are using a force cast `(as!)` rather use an optional cast `(as?)`

Reply:
Looks like you sorted this out 😄